### PR TITLE
Use markdown format for README.

### PR DESCRIPTION
**Pull request by wigging**
Deleted the old `README.rst` and replaced it with a markdown formatted `README.md`. Added sections for "Examples" and "Contributing".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Examples of using PyFR are available in the `examples` directory. Currently
+available examples are:
+
+- 2D Couette flow
+- 2D Euler vortex
+- 2D incompressible cylinder flow
+
+## Contributing
+
+If you would like to contribute to PyFR please follow the steps listed below.
```
Comment (pv101) [inline README.md:23]: Maybe restructure/rephrase as ->

To contribute to PyFR please follow the steps listed below:

1. Fork this repository to your GitHub account
2. Create a new branch in your forked repository
3. Make changes in your new branch
4. Submit your changes by creating a Pull Request to the `develop` branch of the original PyFR repository

Modifications to the `develop` branch are eventually merged into the `master` branch for a new release.

**Comment by pv101** [comment]
Thanks for this. I have made one specific comment regarding the structure and phrasing of the Contributing text. Other than that looks good to go.

File: README.md (hunk 1 above)
Comment (wigging) [reply README.md]: I updated the pull request with these changes.

**Comment by pv101** [comment]
This looks good to me. @FreddieWitherden do you want the two commits squashed into one?

**Pull request merged**